go.mod

```diff
@@ -109,6 +109,7 @@ require (
 	github.com/gobwas/glob v0.2.3 // indirect
 	github.com/gofrs/uuid v4.0.0+incompatible // indirect
 	github.com/gogo/protobuf v1.3.2 // indirect
+	github.com/golang-jwt/jwt/v4 v4.4.3 // indirect
```

Comment:
I'm not sure we are actually able to force this. This is an indirect dependency as a result of pulling in `github.com/distribution/distribution/v3` - which looks like it is pulled in via `helm.sh/helm/v3`.

Unfortunately `github.com/distribution/distribution/v3` has yet to update their golang-jwt version to one with a fix

Reply:
This has been updated and pulled in with https://github.com/operator-framework/operator-sdk/pull/6383/commits/5a865e1aa13b885e5b782b840755e232c1796b09